tests/unit_tests/utils/test_flop_utils.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for flop_utils module."""
+
+import importlib
+
+import pytest
+
+from megatron.bridge.training.utils.flop_utils import num_floating_point_operations
+from megatron.bridge.utils.vocab_utils import calculate_padded_vocab_size
+
+
+class TestFlops:
+    @pytest.mark.parametrize(
+        "model_family,model_config_func_name, seq_length, vocab_size, expected_flops",
+        [
+            ("llama", "llama3_8b_pretrain_config", 8192, 151936, 4.22e14),
```

Comment:
this doesn't work in the unit test suite because we don't supply the HF token. the recipes pull the model configs from HF, and since this is a gated repo, the test is failing.